docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
 title: API Reference
 ---
 
-## Commands
+## Read Model Connector Interface
 
-### Command HTTP API
+The table below lists functions that a custom Read Model's connector should implement.
 
-A command can be sent using HTTP API.
+| Function Name             | Description                                                             |
+| ------------------------- | ----------------------------------------------------------------------- |
+| [connect](#connect)       | Initializes a connection to a storage.                                  |
+| [disconnect](#disconnect) | Closes the storage connection.                                          |
+| [drop](#drop)             | Removes the Read Model's data from storage.                             |
+| [dispose](#dispose)       | Forcefully disposes all unmanaged resources provided by this connector. |
```

Comment:
Firstly, it disposes **of** or it does not - Why do you add 'forcefully'? I've discussed the use of adverbs with you before.

Reply:
dispose **of**

Reply:
ok